doc/python_reference.rst

```diff
@@ -1116,6 +1116,7 @@ Classes:
    UnsupervisedSpatialFilter
    Vectorizer
    ReceptiveField
+   TimeDelayingRidge
```

Comment:
I'd like to expose it because it does have advanced parameters people can choose to use if they know what they're doing